charts/atlantis/templates/statefulset.yaml

```diff
@@ -485,8 +490,10 @@ spec:
             value: {{ .Values.aws.directory }}/config
           {{- end }}
           {{- if .Values.initConfig.enabled }}
+          {{- if not .Values.environment.PATH }}
           - name: PATH
             value: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:{{ .Values.initConfig.sharedDir }}
```

Comment:
The initial paths will be lost if the user specifies a PATH: `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:`. I'm afraid this might be hidden from the chart user. I would say we can either append them (using a named template) or add a comment for the user to know it needs to specify them, when `environment.PATH` is added in either case.

I would prefer the first since usually there is no harm in adding them if you can just prefix with your own. wdyt?

Reply:
Indeed, I would presume user setting a `environment.path` value to include default as well, but that's not awesome I agree.
Agreed with appending paths, no harm there. So if I'm understand correctly we could:
- defined a new value like `extraPath` (or something similar)
- create a new helper that will combine default system path + `initConfig.sharedDir` (if defined) + `extraPath` (if defined)
- explicitly create the environment variable `PATH` with the helper value
This way the `PATH` envvar will always be unique (no duplicate mess) AND will contain at least system required `PATH` + any additional ones ?

Reply:
👍 This works. To maintain backwards compatibility I would set the new `PATH` before `{{- range $key, $value := .Values.environment }}` so we can ensure that users with a `PATH` already set and with `initConfig` disabled won't be affected as it will override the new logic.

Also please:

Reply:
Ok, I've just pushed new logic, If you could double check I'll be grateful 👍